CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @clerkinc/backend-team
```

Comment:
@SokratisVidros could you also give ` @clerkinc/backend-team` team write access for this repo (just like you did in clerk-sdk-ruby).

Reply:
Done